qutip/qobj.py

```diff
@@ -1003,9 +979,17 @@ def norm(self, norm=None, sparse=False, tol=0, maxiter=100000):
 
         Returns
         -------
-        norm : float
+        float
             The requested norm of the operator or state quantum object.
 
+        Raises
+        -------
+        ValueError : For matrices, norm must be 'tr', 'fro', 'one', or 'max'
+            When norm chosen is not from default methods of operators.
+
+        ValueError : For vectors, norm must be 'l2', or 'max'
+            When norm chosen is not from default methods of bra/kets.
```

Comment:
you don't need to have a "Raises" section for "obvious" errors like "passed a completely invalid parameter"